src/network/api/get-distribution-list.ts

```diff
@@ -28,7 +29,7 @@ export interface GetDistributionListResponse extends GenericSoapPayload<typeof N
 		_attrs?: {
 			displayName?: string;
 			description?: string;
-			zimbraHideInGal?: 'TRUE' | 'FALSE';
+			zimbraHideInGal: ZimbraHideInGalType;
```

Comment:
Set the field as optional with the `?` signature, so that we don't have to specify it in handlers (also, the field might not be present in the network response I think)